### ISSUE TITLE
Please don't use version ranges for guava (and other libraries)

### ISSUE THREAD

**Issue by DonatasC** [eclipse/xtext-xtend#586]
Please please could you specify particular guava version in your org.eclipse.xtend.maven.parent. Now you have:
<guava.version>**[14.0,22)**</guava.version>
and this results in:
1. Once every day Maven must check whether no new guava build matching the range appeared. As other Eclipse projects use ranges too, my maven build time increases from 20 seconds to 3 minutes... (on the second time Maven uses cached results and builds become OK...)
2. As "newest" (by Maven rules) is guava "22.0-SNAPSHOT" (and this is CRAZY!!!), I'm randomly (on some days - consistently) getting:

[ERROR] Failed to execute goal org.eclipse.xtend:xtend-maven-plugin:2.15.0:compile (default) on project ndd-meta-model: Execution default of goal org.eclipse.xtend:xtend-maven-plugin:2.15.0:compile failed: Plugin org.eclipse.xtend:xtend-maven-plugin:2.15.0 or one of its dependencies could not be resolved: **Could not find artifact com.google.guava:guava:jar:22.0-SNAPSHOT** -> [Help 1]
[11:00:32][Step 2/2] [ERROR] 

Could you please instead of [14.0,22) specify released version of guava? 21? or any other that suits you?

**Comment by kthoms** [comment]
You could easily avoid this by specifying an explicit dependency in the dependencies section of the xtend-maven-plugin.
We'll discuss this and decide whether we'll change that or not.

**Comment by DonatasC** [comment]
Yes, I have explicit dependency, and this does not help. That explicit dependency is used during runtime, but during resolving phase maven is still fetching all POM's mentioned in your parent (to build pom tree).
So this does NOT help.

**Comment by DonatasC** [comment]
And I strongly believe that released product should not depend on SNAPSHOT versions, current version range [14, 22) actually depends on 22-SNAPSHOT.

**Comment by cdietrich** [comment]
@DonatasC how does your repo config look like?

**Comment by DonatasC** [comment]
This is corporate environment, all devs have standard settings.xml that has <mirror> configuration to redirect all remote repos to company local Artifactory. 
Everything is working, builds finish OK. I'm complaining that during build maven tries to fetch guava:22.0-SNAPSHOT:pom to build dependency tree.
Try it yourself:
    mvn dependency:resolve-plugins
and observe the output - maven will try to "scan" guava:22.0-SNAPSHOT.

I don't really understand what is the problem of specifying guava version 21.0 release for example?

**Comment by DonatasC** [comment]
Even if I specify excludes:
                ```
<plugin>
                    <groupId>org.eclipse.xtend</groupId>
                    <artifactId>xtend-maven-plugin</artifactId>
                    <version>${xtext-version}</version>
                    <dependencies>
                        <dependency>
                            <groupId>org.eclipse.xtext</groupId>
                            <artifactId>org.eclipse.xtext.xbase.lib</artifactId>
                            <version>${xtext-version}</version>
                            <exclusions>
                                <exclusion>
                                    <groupId>com.google.guava</groupId>
                                    <artifactId>guava</artifactId>
                                </exclusion>
                            </exclusions>
                        </dependency>
                        <dependency>
                            <groupId>com.google.guava</groupId>
                            <artifactId>guava</artifactId>
                            <version>26.0-jre</version>
                        </dependency>
                    </dependencies>
                </plugin>
```

**mvn clean install -U** still gives:
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/maven-metadata.xml
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/maven-metadata.xml (4.2 kB at 16 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/**22.0-SNAPSHOT**/maven-metadata.xml
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/22.0-SNAPSHOT/maven-metadata.xml (1.2 kB at 9.3 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/22.0-SNAPSHOT/maven-metadata.xml
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/22.0-SNAPSHOT/maven-metadata.xml (607 B at 4.5 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/org/eclipse/platform/org.eclipse.osgi/maven-metadata.xml

And I'm having hard time fighting our Artifactory admin not to restrict SNAPSHOT's for released product versions... :(

**Comment by cdietrich** [comment]
So it depends cause you have it in your local corporate repo

**Comment by DonatasC** [comment]
Well I went to my local .m2 and removed all guava folders. 
**mvn clean install -U** gave this nightmare (it scanned everything in the range 14-22):
`
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/maven-metadata.xml
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/maven-metadata.xml (4.2 kB at 20 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/14.0/guava-14.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/14.0/guava-14.0.pom (5.4 kB at 42 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/14.0/guava-parent-14.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/14.0/guava-parent-14.0.pom (2.6 kB at 22 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/14.0.1/guava-14.0.1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/14.0.1/guava-14.0.1.pom (5.4 kB at 47 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/14.0.1/guava-parent-14.0.1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/14.0.1/guava-parent-14.0.1.pom (2.6 kB at 23 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/14.0.1.redhat-1/guava-14.0.1.redhat-1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/14.0.1.redhat-1/guava-14.0.1.redhat-1.pom (5.3 kB at 21 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/14.0.1.redhat-1/guava-parent-14.0.1.redhat-1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/14.0.1.redhat-1/guava-parent-14.0.1.redhat-1.pom (5.9 kB at 52 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/15.0-rc1/guava-15.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/15.0-rc1/guava-15.0-rc1.pom (6.3 kB at 52 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/15.0-rc1/guava-parent-15.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/15.0-rc1/guava-parent-15.0-rc1.pom (7.3 kB at 64 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/15.0/guava-15.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/15.0/guava-15.0.pom (6.3 kB at 58 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/15.0/guava-parent-15.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/15.0/guava-parent-15.0.pom (7.3 kB at 65 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/16.0-rc1/guava-16.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/16.0-rc1/guava-16.0-rc1.pom (6.1 kB at 53 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/16.0-rc1/guava-parent-16.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/16.0-rc1/guava-parent-16.0-rc1.pom (7.3 kB at 63 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/16.0/guava-16.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/16.0/guava-16.0.pom (6.1 kB at 60 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/16.0/guava-parent-16.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/16.0/guava-parent-16.0.pom (7.3 kB at 67 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/16.0.1/guava-16.0.1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/16.0.1/guava-16.0.1.pom (6.1 kB at 53 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/16.0.1/guava-parent-16.0.1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/16.0.1/guava-parent-16.0.1.pom (7.3 kB at 69 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/16.0.1.redhat-3/guava-16.0.1.redhat-3.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/16.0.1.redhat-3/guava-16.0.1.redhat-3.pom (6.0 kB at 50 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/16.0.1.redhat-3/guava-parent-16.0.1.redhat-3.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/16.0.1.redhat-3/guava-parent-16.0.1.redhat-3.pom (9.2 kB at 81 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/17.0-rc1/guava-17.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/17.0-rc1/guava-17.0-rc1.pom (5.6 kB at 48 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/17.0-rc1/guava-parent-17.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/17.0-rc1/guava-parent-17.0-rc1.pom (7.8 kB at 73 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/17.0-rc2/guava-17.0-rc2.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/17.0-rc2/guava-17.0-rc2.pom (5.6 kB at 55 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/17.0-rc2/guava-parent-17.0-rc2.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/17.0-rc2/guava-parent-17.0-rc2.pom (7.8 kB at 66 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/17.0/guava-17.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/17.0/guava-17.0.pom (5.6 kB at 51 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/17.0/guava-parent-17.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/17.0/guava-parent-17.0.pom (7.8 kB at 52 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/18.0-rc1/guava-18.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/18.0-rc1/guava-18.0-rc1.pom (5.7 kB at 52 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/18.0-rc1/guava-parent-18.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/18.0-rc1/guava-parent-18.0-rc1.pom (7.7 kB at 73 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/18.0-rc2/guava-18.0-rc2.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/18.0-rc2/guava-18.0-rc2.pom (5.7 kB at 51 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/18.0-rc2/guava-parent-18.0-rc2.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/18.0-rc2/guava-parent-18.0-rc2.pom (7.7 kB at 75 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/18.0/guava-18.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/18.0/guava-18.0.pom (5.7 kB at 52 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/18.0/guava-parent-18.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/18.0/guava-parent-18.0.pom (7.7 kB at 75 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/18.0.0.redhat-1/guava-18.0.0.redhat-1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/18.0.0.redhat-1/guava-18.0.0.redhat-1.pom (5.6 kB at 47 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/18.0.0.redhat-1/guava-parent-18.0.0.redhat-1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/18.0.0.redhat-1/guava-parent-18.0.0.redhat-1.pom (9.6 kB at 81 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0-rc1/guava-19.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0-rc1/guava-19.0-rc1.pom (6.8 kB at 63 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0-rc1/guava-parent-19.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0-rc1/guava-parent-19.0-rc1.pom (9.9 kB at 92 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0-rc2/guava-19.0-rc2.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0-rc2/guava-19.0-rc2.pom (6.8 kB at 64 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0-rc2/guava-parent-19.0-rc2.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0-rc2/guava-parent-19.0-rc2.pom (9.9 kB at 48 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0-rc3/guava-19.0-rc3.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0-rc3/guava-19.0-rc3.pom (6.8 kB at 53 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0-rc3/guava-parent-19.0-rc3.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0-rc3/guava-parent-19.0-rc3.pom (9.9 kB at 91 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0/guava-19.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0/guava-19.0.pom (6.8 kB at 47 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0/guava-parent-19.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0/guava-parent-19.0.pom (9.9 kB at 90 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0.0.jbossorg-1/guava-19.0.0.jbossorg-1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0.0.jbossorg-1/guava-19.0.0.jbossorg-1.pom (6.9 kB at 58 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0.0.jbossorg-1/guava-parent-19.0.0.jbossorg-1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0.0.jbossorg-1/guava-parent-19.0.0.jbossorg-1.pom (9.9 kB at 82 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0.0.jbossorg-2/guava-19.0.0.jbossorg-2.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0.0.jbossorg-2/guava-19.0.0.jbossorg-2.pom (6.8 kB at 31 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0.0.jbossorg-2/guava-parent-19.0.0.jbossorg-2.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0.0.jbossorg-2/guava-parent-19.0.0.jbossorg-2.pom (9.9 kB at 88 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0.0.redhat-1/guava-19.0.0.redhat-1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0.0.redhat-1/guava-19.0.0.redhat-1.pom (6.8 kB at 59 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0.0.redhat-1/guava-parent-19.0.0.redhat-1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/19.0.0.redhat-1/guava-parent-19.0.0.redhat-1.pom (12 kB at 107 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0.20150826/guava-19.0.20150826.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/19.0.20150826/guava-19.0.20150826.pom (465 B at 4.2 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/20.0-rc1/guava-20.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/20.0-rc1/guava-20.0-rc1.pom (6.8 kB at 65 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/20.0-rc1/guava-parent-20.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/20.0-rc1/guava-parent-20.0-rc1.pom (9.9 kB at 80 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/20.0/guava-20.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/20.0/guava-20.0.pom (6.8 kB at 46 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/20.0/guava-parent-20.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/20.0/guava-parent-20.0.pom (9.9 kB at 82 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/20.0-hal/guava-20.0-hal.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/20.0-hal/guava-20.0-hal.pom (6.8 kB at 57 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/20.0-hal/guava-parent-20.0-hal.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/20.0-hal/guava-parent-20.0-hal.pom (10 kB at 90 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/20.0.0.redhat-1/guava-20.0.0.redhat-1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/20.0.0.redhat-1/guava-20.0.0.redhat-1.pom (6.4 kB at 56 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/20.0.0.redhat-1/guava-parent-20.0.0.redhat-1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/20.0.0.redhat-1/guava-parent-20.0.0.redhat-1.pom (11 kB at 91 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/21.0-rc1/guava-21.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/21.0-rc1/guava-21.0-rc1.pom (7.0 kB at 48 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/21.0-rc1/guava-parent-21.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/21.0-rc1/guava-parent-21.0-rc1.pom (9.7 kB at 81 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/21.0-rc2/guava-21.0-rc2.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/21.0-rc2/guava-21.0-rc2.pom (7.0 kB at 65 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/21.0-rc2/guava-parent-21.0-rc2.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/21.0-rc2/guava-parent-21.0-rc2.pom (9.7 kB at 86 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/21.0/guava-21.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/21.0/guava-21.0.pom (7.0 kB at 51 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/21.0/guava-parent-21.0.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/21.0/guava-parent-21.0.pom (9.7 kB at 86 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/22.0-rc1/guava-22.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/22.0-rc1/guava-22.0-rc1.pom (5.7 kB at 42 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/22.0-rc1/guava-parent-22.0-rc1.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/22.0-rc1/guava-parent-22.0-rc1.pom (8.9 kB at 83 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/22.0-rc1-android/guava-22.0-rc1-android.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/22.0-rc1-android/guava-22.0-rc1-android.pom (5.7 kB at 52 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/22.0-rc1-android/guava-parent-22.0-rc1-android.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/22.0-rc1-android/guava-parent-22.0-rc1-android.pom (8.6 kB at 79 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/22.0-SNAPSHOT/maven-metadata.xml
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/22.0-SNAPSHOT/maven-metadata.xml (1.2 kB at 10 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/22.0-SNAPSHOT/guava-22.0-20170523.165200-168.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava/22.0-SNAPSHOT/guava-22.0-20170523.165200-168.pom (5.7 kB at 2.9 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/22.0-SNAPSHOT/maven-metadata.xml
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/22.0-SNAPSHOT/maven-metadata.xml (607 B at 5.2 kB/s)
Downloading from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/22.0-SNAPSHOT/guava-parent-22.0-20170523.165107-168.pom
Downloaded from repo: http://svnprod.corp.ad.local/artifactory/repo/com/google/guava/guava-parent/22.0-SNAPSHOT/guava-parent-22.0-20170523.165107-168.pom (8.9 kB at 13 kB/s)
`
Guys, you cannot be serious about this... this is some kind of comedy...

**Comment by tivervac** [comment]
@cdietrich That's what I meant indeed

**Comment by cdietrich** [comment]
cc @kthoms you are on this anyway

**Comment by kthoms** [comment]
Xtext 2.17 will use a Maven BOM (see eclipse/xtext#1249) and Guava will be pinned to version 21.0 by it. This will avoid these nasty range checks and improves build speed.

**Comment by cdietrich** [comment]
thus this is fixed in 2.17
